### PR TITLE
Add functionality for quotes (Holidays, Hotels, Flights)

### PR DESCRIPTION
**Pull request by dav-armour**
Created models and schemas
Also created seeds for quotes

### CODE REVIEW

File: database/schemas/holiday_quote_schema.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const { Schema } = require("mongoose");
+
+const HolidayQuoteSchema = new Schema({
+  budget_tier: {
+    type: String,
+    enum: ["budget", "mid-range", "luxury"],
+    required: true
```
Comment (natashamunasinghe) [inline database/schemas/holiday_quote_schema.js]: Consider rewording " mid-range"

File: database/schemas/hotel_quote_schema.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    required: true
+  },
+  num_stars: {
+    type: Number,
+    required: true,
+    min: 1,
+    max: 5
+  }
+});
+
```
[the remark this answers is not in the document]
Comment (dav-armour) [reply database/schemas/hotel_quote_schema.js]: Not needed can calculate based on dates

File: database/schemas/flight_quote_schema.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+const { Schema } = require("mongoose");
+
+const FlightQuoteSchema = new Schema({
+  seat_type: {
+    type: String,
+    enum: ["economy", "premium economy", "business", "first class"],
+    required: true
+  },
```
[the remark this answers is not in the document]
Comment (dav-armour) [reply database/schemas/flight_quote_schema.js]: Have added ticket_type field

File: database/schemas/quote_schema.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+      type: Boolean,
+      required: true
+    },
+    user: QuoteUserSchema,
+    comments: {
+      type: String
+    }
+  },
+  {
+    timestamps: {},
```
[the remark this answers is not in the document]
Comment (dav-armour) [reply database/schemas/quote_schema.js]: Added status: new, researching, pending, finalized, declined

File: database/schemas/holiday_quote_schema.js (hunk 1 above)
Comment (dav-armour) [reply database/schemas/holiday_quote_schema.js]: Leaving for now

**Pull request merged**